packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+		"defaultValue": 0
+	},
+	"power_1_periodic_report": {
+		// param 7
+		"label": "Power 1 Periodic Report",
+		"description": "Used to set the time in seconds between sending Power 1 METER REPORTs to the Lifeline.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 3600
+	},
+	"power_1_change_report": {
+		// param 8
+		"label": "Power 1 Change Report",
+		"description": "Used to set the power in Watts that a reported reading of Power 1 must change by since the last reported value before a new METER REPORT is sent to the Lifeline.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 10
+	},
+	"power_1_renotification_delay": {
+		// param 9
+		"label": "Power 1 Renotification Delay",
+		"description": "Used to set the time in seconds between sending repeated Power 1 METER REPORTs to the Lifeline while in a Power 1 Low or High state.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_entry_set_value": {
+		// param 10
+		"label": "Power 1 High Entry SET value",
+		"description": "BASIC SET value sent to associated nodes on entry into Power 1 High state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 255
+	},
+	"power_1_high_exit_set_value": {
+		// param 11
+		"label": "Power 1 High Exit SET value",
+		"description": "BASIC SET value sent to associated nodes on exit from Power 1 High state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_low_entry_set_value": {
+		// param 12
+		"label": "Power 1 Low Entry SET Value",
+		"description": "BASIC SET value sent to associated nodes on entry into Power 1 Low state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 255
+	},
+	"power_1_low_exit_set_value": {
+		// param 13
+		"label": "Power 1 Low Exit SET Value",
+		"description": "BASIC SET value sent to associated nodes on exit from Power 1 Low state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_actions_control_associations_on_entry": {
+		// param 14b0
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Power 1 High Actions: CONTROL ASSOCIATIONS ON ENTRY",
+		"description": "If set, send BASIC SET with “Power 1 High Entry SET Value” to “Power 1 High” association group on Power 1 High state entry."
+	},
+	"power_1_high_actions_control_associations_on_exit": {
+		// param 14b1
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Power 1 High Actions: CONTROL ASSOCIATIONS ON EXIT",
+		"description": "If set, send BASIC SET with “Power 1 High Exit SET Value” to “Power 1 High” association group on Power 1 High state exit."
```

```suggestion
		"label": "Load High Power Actions: Basic Set to Associations (Exit)",
```
etc...

Try to formulate labels in a way that makes descriptions obsolete, and avoid UPPERCASE